### PR TITLE
Feature/group editing swipe

### PR DESCRIPTION
**Pull request by AwesomeFlax**
image
image
image
image
#72 #40

### CODE REVIEW

File: src/features/groups/Edit/index.tsx [hunk 1]
```diff
@@ -51,27 +37,21 @@ const Edit = () => {
     <PageWrapper>
       <Header
         text={t(groupId ? 'updateGroup' : 'createGroup')}
-        isLoading={isLoading}
```
Comment (doskachok) [inline src/features/groups/Edit/index.tsx:54]: Why did you remove isLoading?

Comment (AwesomeFlax) [reply src/features/groups/Edit/index.tsx:54]: It's back!

File: src/features/groups/Edit/index.tsx [hunk 2]
```diff
@@ -13,65 +13,76 @@ import Members from '../Members';
...
-    setMode(mode => mode === GroupEditMode.GeneralInfo ? GroupEditMode.Members : GroupEditMode.GeneralInfo);
-  }
 
   useEffect(() => {
     if (groupId) {
       getGroup(groupId);
     }
   }, [groupId, getGroup]);
 
+  let sliderRef = useRef<Slider>(null);
```
Comment (doskachok) [inline src/features/groups/Edit/index.tsx]: Could you please, make sliderRef const?

File: src/features/groups/Edit/index.tsx [hunk 3]
```diff
@@ -13,65 +13,76 @@ import Members from '../Members';
...
+
+  var settings = {
```
Comment (doskachok) [inline src/features/groups/Edit/index.tsx]: Could you please, make settings const?

File: src/features/groups/Edit/index.tsx [hunk 4]
```diff
@@ -13,65 +13,76 @@ import Members from '../Members';
...
+            isActive={activeSliderItem === item} 
+            onClick={() => { setActiveSliderItem(item); sliderRef.current?.slickGoTo(idx); }}
+          >
+            {t(item + 'SliderItem')}
+          </SliderNavItem>
+          )
+      })}
+    </SliderItemsWrapper>
+
+  }
```
Comment (doskachok) [inline src/features/groups/Edit/index.tsx]: Would it be possible to move sliderPagesList function outside of the component to avoid its retendering each time?

File: src/features/groups/Edit/index.tsx [hunk 5]
```diff
@@ -13,65 +13,76 @@ import Members from '../Members';
...
-  GeneralInfo,
-  Members,
-}
+import Slider from "react-slick";
 
 const Edit = () => {
   const { t } = useTranslation('groups');
   const navigate = useNavigate();
   const { groupId } = useParams();
+  const sliderItems = groupId ? ["mainInfo", "members", "statistics"] : ["mainInfo"];
```
Comment (doskachok) [inline src/features/groups/Edit/index.tsx]: Would it make sense to replace these strings with enum?

**Review by doskachok — CHANGES_REQUESTED** [review]

File: src/features/groups/Edit/index.tsx (hunk 5 above)
Comment (AwesomeFlax) [reply src/features/groups/Edit/index.tsx]: Yeah, it definitely would be better

File: src/features/groups/Edit/index.tsx (hunk 4 above)
Comment (AwesomeFlax) [reply src/features/groups/Edit/index.tsx]: It’d be better to do that, yes this way, yes.
